modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -33,19 +33,46 @@ overlay_size=$(getarg rd.live.overlay.size=)
 getargbool 0 rd.live.overlay.thin && thin_snapshot="yes"
 getargbool 0 rd.live.overlay.overlayfs && overlayfs="yes"
 
+# Take a path to a disk label and return the parent disk if it is a partition
```

Comment:
Would it make sense to add this function to dracut-lib.sh instead to make it accessible in other modules as well as needed ?

Reply:
I'm not sure. I think it's probably of limited usefulness outside of dmsquash-live.